playbooks/technology_radar/data.csv

```diff
@@ -0,0 +1,48 @@
+name,ring,quadrant,isNew,description
+Babel,adopt,tools,TRUE,"This is the description. You can use basic html such as the <strong>strong tag to emphasise keywords and phrases</strong> and insert <a href=""https://www.thoughtworks.com"">anchor links to documentation and referance material</a>."
```

Comment:
When is something new? Babel, Travis, GraphQL etc have all been adopted long before eg OpsGenie.